features/flaw_detail.feature

```diff
@@ -110,3 +110,8 @@ Feature: Flaw detail testing
       Given I go to an embargoed flaw detail page
       When I update the embargoed flaw with a future public date
       Then The embargoed flaw is updated
+
+    Scenario: Add new affect to flaw
+      Given I go to a public flaw detail page
+      When I add a new affect with valid data
+      Then The affect is added
```

Comment:
Meet "Flaw validation failed before submission
20 seconds ago
affects/3/cvss_scores/0/affect: Required
 affects/3/cvss_scores/0/comment: Required" error info. Is this a bug? Using flaw "CVE-2023-2023"